packages/frontend/core/src/providers/session-provider.tsx

```diff
@@ -22,14 +21,14 @@ const SessionDefence = (props: PropsWithChildren) => {
   const prevSession = useRef<ReturnType<typeof useSession>>();
   const [sessionInAtom, setSession] = useAtom(sessionAtom);
   const pushNotification = useSetAtom(pushNotificationAtom);
-  const refreshMetadata = useSetAtom(refreshRootMetadataAtom);
   const onceSignedInEvents = useOnceSignedInEvents();
   const t = useAFFiNEI18N();
 
   const refreshAfterSignedInEvents = useAsyncCallback(async () => {
     await onceSignedInEvents();
-    refreshMetadata();
-  }, [onceSignedInEvents, refreshMetadata]);
+    // TODO: refresh metadata
+    // refreshMetadata();
```

Comment:
It seems the workspace list does not refresh after logged in, which is critical.
Do you have any idea on fixing this issue?

Reply:
Oh, I missed this todo.
I'm fix this by sending a `affine-cloud-workspace-changed` event via `BroadcastChannel`.